Assets/Scripts/Drinkables/UpsideDownBeer.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
```

Comment:
If you guys are down, I can also inverse the camera commands in refactory PR since I want to add an options in the menu to set your camera axis anyways

Reply:
If no one else is doing it already i don't see why not